Algorithm.CSharp/BasicTemplateOptionsFrameworkAlgorithm.cs

```diff
@@ -45,7 +44,7 @@ public override void Initialize()
             // set framework models
             SetUniverseSelection(new EarliestExpiringWeeklyAtTheMoneyPutOptionUniverseSelectionModel(SelectOptionChainSymbols));
             SetAlpha(new ConstantOptionContractAlphaModel(InsightType.Price, InsightDirection.Up, TimeSpan.FromHours(0.5)));
-            SetPortfolioConstruction(new SingleSharePortofioConstructionModel());
+            SetPortfolioConstruction(new SingleSharePortfolioConstructionModel());
```

Comment:
Perhaps we should make this algorithm implement `IRegressionAlgorithmDefintion`?